build_duet.tool

```diff
@@ -165,5 +165,5 @@ else
   export SELFPKG
   export NO_ARCHIVES
 
-  src=$(curl -Lfs https://raw.githubusercontent.com/acidanthera/ocbuild/master/efibuild.sh) && eval "$src" || exit 1
+  src=$(curl -Lfs https://raw.githubusercontent.com/acidanthera/ocbuild/merge-min_size-assert/efibuild.sh) && eval "$src" || exit 1
```

Comment:
Change from `master` in this line I believe was an error last time it occurred. Is this intended?

Reply:
It is only needed for CI, these changes are not supposed to be included in the merge.